udata/core/dataset/models.py

```diff
@@ -78,7 +77,7 @@ def get_json_ld_extra(key, value):
     }
 
 
-class HarvestDatasetMetadata(DynamicEmbeddedDocument):
```

Comment:
Why this change in this PR?

Reply:
Ah sorry, I hadn't published my comment:
I don't think we want to bother with the DynamicEmbeddedDocument pattern since we don't have any other harvest-related plugin. Removed it in https://github.com/opendatateam/udata/pull/3335/commits/18d8101995d53ac7738b732474fd913d50b452e1.

Reply:
Yes but not really related to the changes in this PR? :-(

Reply:
But I can create another PR with these modifications indeed, it will be needed for this PR though since dynamic documents would [make tests fail](https://app.circleci.com/pipelines/github/opendatateam/udata/6982/workflows/b5d72cae-7de9-4a94-8478-918cd39e9bbf/jobs/36896?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary).

Reply:
Oh, so there is a link with this PR, so I'm fine with it being here :-)

Reply:
Woops, moved it here: https://github.com/opendatateam/udata/pull/3396 :p